src/sentry/workflow_engine/typings/notification_action.py

```diff
@@ -352,3 +398,23 @@ class AzureDevOpsDataBlob(TicketDataBlob):
 
     project: str = ""
     work_item_type: str = ""
+
+
+@dataclass
+class JiraDataBlob(TicketDataBlob):
+    """
+    JiraDataBlob represents the data blob for a Jira ticket creation action.
+    Includes required fields and supports dynamic custom fields.
+    """
+
+    project: str = ""
+    issuetype: str = ""
+    assignee: str = ""
+    reporter: str = ""
+    description: str = ""
+    # Optional standard fields
+    labels: str = ""
+    parent: str = ""
+    priority: str = ""
```

Comment:
Where did this list of fields come from? Looking through the jira issue metadata on our test projects, it isn't guaranteed that an issue contains all of these, and may contain additional fields outside of these 😭

Reply:
Following up on this, here are the minimal set of fields we inherently support, but they may still be optional in some cases:
https://github.com/getsentry/sentry/blob/992890e31747136d4845992b90fa6f15d11705c6/src/sentry/integrations/jira/integration.py#L343

Both `get_persisted_default_config_fields` and `get_persisted_user_default_config_fields` should probably be the minimal set of supported fields, the rest can be dumped into the dynamic form fields to be flattened later.